packages/components/src/InputEmail/InputEmail.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { FormField, FormFieldProps } from "../FormField";
+
+type InputEmailProps = Pick<
```

Comment:
Since we are using the `value` from `FormFieldProps`, it can still be a number with is an invalid type for email.

Reply:
Hey @hosseintalebi!  Could you elaborate a little bit?  We do have a regex in place to check if the email is valid